_maps/modularmaps/EORG/de_inferno.dmm

```diff
@@ -1206,7 +1206,7 @@
 	name = "Blocker"
 	},
 /turf/closed/wall/wood{
-	soft_armor = list("melee" = 1000, "bullet" = 1000, "laser" = 1000, "energy" = 1000, "bomb" = 1000, "bio" = 1000, "rad" = 1000, "fire" = 1000, "acid" = 1000)
+	soft_armor = list(MELEE = 1000, BULLET = 1000, LASER = 1000, ENERGY = 1000, BOMB = 1000, BIO = 1000, "rad" = 1000, FIRE = 1000, ACID = 1000)
```

Comment:
who

Reply:
![bild](https://user-images.githubusercontent.com/63870402/192084537-0db30bf6-409a-400e-9b1d-9fa08ec09f88.png)

Reply:
this one too